filament/src/materials/tonemapping.mat

```diff
@@ -0,0 +1,57 @@
+material {
+    name : tonemap,
+    parameters : [
+        {
+            type : sampler2d,
+            name : colorBuffer,
+            precision: high
+        },
+        {
+            type : int,
+            name : dithering
+        }
+    ],
+    depthWrite : false,
+    depthCulling : false,
+    culling: none,
+    domain: postprocess
+}
+
+fragment {
+
+#include "../../../shaders/src/tone_mapping.fs"
```

Comment:
I think ideally these can be moved out of the `shaders` library and live next to this `tonemapping.mat` file, since they don't provide any user API functions.

Reply:
I would prefer to keep all the shaders in the same spot. shaders/ contains more than just public APIs.

Reply:
Sure, makes sense. After deprecating "old" post-process materials they can at least be removed from `libshaders`